### ISSUE TITLE
Looking for Suggestion

### ISSUE THREAD

**Issue by pragazv** [Jyouhou/SceneTextPapers#2]
Hi, 
Thanks to share your impressive collections in Scene Text Papers. 
I would like to get some suggestions for my queries
can u suggest which model is the best scene text detection model. 
I would like to run the custom trained model prediction in a mobile devices. so, I am trying to find a
model with small size
model with high accuracy which gives result for most of the problems
which run in mobile devices with prediction in millisecond/frame result. 

currently i tried with pre-trained models like
East model -argman - Resnet 50
East model - Kurapan - Resnet 50
Advance east -VGG16
CTPN -VGG16
and yet to try CRNN
out of this CTPN shows high accuracy but taking 2.5 sec without any gpu in laptop. 
and East argman took 1.6 sec with low accuracy compared with CTPN model (gpu=0)
moving this to mobile device increases the delay in detection. 

if you could suggest me to get some progress that will be really supportive . 
suggestion can be anything like customizing model, changing the loss function, optimization, merging the major features of diff model, or any new model with best performance as per the requirements. or it can be anything new to try out .

**Comment by longshangbang** [comment]
You can try replacing the backbones, e.g. VGG/ResNet, with those light-weight models. That should be an effective way for speedup.

As for the algorithm, it depends on the scenario, whether it's mainly long text, curved text, or small text. :)
